### ISSUE TITLE
Add a DID method for this?

### ISSUE THREAD

**Issue by tyleryasaka** [IBM/ipfs-social-proof#17]
For context, I'm referring to the Decentralized Identifiers (DIDs) spec. https://w3c-ccg.github.io/did-spec/

Probably a bit early, but I think it's worth mentioning: would be great if this could be defined as a formal DID method.

I would be glad to help make this happen.

One example of a DID method: https://github.com/WebOfTrustInfo/rebooting-the-web-of-trust-spring2018/blob/master/topics-and-advance-readings/DID-Method-erc725.md

Current list of DID methods: https://w3c-ccg.github.io/did-method-registry/

**Comment by daviddahl** [comment]
Yes. supporting the DID spec is part of the plan, thank you for articulating this here.

 I have just been getting versed in much of the DID spec work and various efforts around it. I have a very rudimentary data object right now that collects the hashed pub key, 'peerId', arbitrary name`handle`, public key and a set of signed proofs are locally stored, but it does need to be interoperable with other systems.

**Comment by daviddahl** [comment]
Wanted to reference https://github.com/jonnycrunch/ipid here as the general concept to follow
